extensions/notion/src/components/PageListItem.tsx

```diff
@@ -121,9 +121,21 @@ export function PageListItem({
     <Action title="Open in Notion" icon="notion-logo.png" onAction={() => handleOnOpenPage(page, setRecentPage)} />
   );
 
+  const { open_in } = getPreferenceValues<Preferences>();
+  const otherNotionActionTitle = open_in === "web" ? "Open in App" : "Open in Browser";
+  const otherNotionAction = open_in === "web" ? "app" : "web";
+  const openInNotionOtherAction = (
+    <Action
+      title={otherNotionActionTitle}
+      icon="notion-logo.png"
```

Comment:
It'd be nice that opening in the browser displays an `Icon.Globe` to make it clear the page will open in the web browser.

Reply:
Good idea, done